pyproject.toml

```diff
@@ -13,10 +13,13 @@ dependencies = [
     "fal",
     "faster_whisper",
     "google-cloud-texttospeech",
+    "numpy",
     "openai",
     "Pillow",
     "pyht",
     "python-dotenv",
+    "torch",
+    "pyaudio",
```

Comment:
pyaudio introduces an additional outside-of-pypi dependency (you've gotta `brew insall portaudio` on macos). I'm a little on-the-fence about requiring this for the "standard" SDK. Do we think everyone will want VAD, so it's okay to take on this dependency?

Reply:
I think VAD is going to be pretty instrumental to anybody wanting to do conversations, but I hear your concern. Should we do some kind of [conditional import](https://stackoverflow.com/questions/3496592/conditional-import-of-modules-in-python) and add a note in the README about the macOS dependency?

Reply:
why is pyaudio required?